settings.gradle.kts

```
@@ -29,3 +29,4 @@ include(":core:presentation:ui")
 include(":core:database")
 include(":core:domain")
 include(":core:data")
+include(":booking")
```

Para este caso solo se accede a los componentes de UI, por este motivo solo debe hacer referencias al submodulo especifico.

```suggestion
include(":booking:presentation")
```

recuerda que primero hay que crear el submodulo "presentation"